src/site/content/en/blog/image-policies/index.md

```diff
@@ -0,0 +1,265 @@
+
+I get similar results if I lower only the width or the height.
+
+![Resized width](resize-width.png) ![Resized height](resized-height.png)
+
+#### How to use
+
+If you are new to feature policy, please check out [Introduction to Feature
+Policy](https://developers.google.com/web/updates/2018/06/feature-policy) for
+more details.
+
+To summarize, `oversized-images` policy can be specified through either:
+
+* `Feature-Policy` HTTP header
+* `<iframe>` `allow` attribute
+
+To declare the `oversized-images` policy, you need to provide:
+
+* The feature name, `oversized-images` (Required)
+* A list of origins (Optional)
+* The threshold values (i.e., the downscaling ratio X) for the origins, specified in
+  parenthesis (Optional)
+
+We recommend a downscaling ratio of 2.0 or lower. Consider using responsive
+images with different resolutions to best serve images on various screen sizes,
+resolutions, etc..
+
+#### More examples
+
+`Feature-Policy: oversized-images *(2.0)`
+
+The policy is enforced on all origins with a threshold value of 2.0. Any
+`<img>` element with an image whose downscaling ratio that is greater than
+2.0 is disallowed and will be replaced with a placeholder image.
+
+`Feature-Policy: oversized-images *(inf) 'self'(1.5)`
+
+The policy is enforced on the site's origin with a threshold value of 1.5.
+`<img>` elements in top-level browsing contexts and same origin nested
+browsing contexts will only render normally if the downscaling ratio is less
+than or equal to 1.5. `<img>` elements everywhere else will render normally.
+
+### unoptimized-{lossy,lossless}-images
+The `unoptimized-lossy-images`, `unoptimized-lossless-images`,
+`unoptimized-lossless-images-strict` feature policies restrict the file
+size of an image in relation to its intrinsic resolution:
+
+<dl>
+  <dt><code>unoptimized-lossy-images</code></dt>
+  <dd>Lossy formats should not exceed a byte-per-pixel ratio of X, with a fixed <strong>1KB</strong> overhead allowance. For a W x H image, the file size threshold is calculated as W x H x X + 1024.</dd>
+  <dt><code>unoptimized-lossless-images</code></dt>
+  <dd>Lossless formats should not exceed a byte-per-pixel ratio of X, with a fixed <strong>10KB</strong> overhead allowance. For a W x H image, the file size threshold is calculated as W x H x X + 10240.</dd>
+  <dt><code>unoptimized-lossless-images-strict</code></dt>
+  <dd>Lossless formats should not exceed a byte-per-pixel ratio of X, with a fixed <strong>1KB</strong> overhead allowance. For a W x H image, the file size threshold is calculated as W x H x X + 1024.</dd>
+</dl>
+
+When a document uses any of these policies, any `<img>` element violating
+the constraint will be replaced with a placeholder image.
+
+#### Why?
+
+The larger the download size is, the longer it takes for an image to load. The
+file size should be kept as small as possible when optimizing an image:
+stripping metadata, picking a good image format, using image compression, etc..
+
```

Comment:
Can we add Can we add here:
Read [Use Imagemin to compress images](https://web.dev/use-imagemin-to-compress-images/) and [Use WebP images](https://web.dev/serve-images-webp/) on how to optimize your images.

Reply:
Done.